src/Sarif.Viewer.VisualStudio.UnitTests/ImportanceToForegroundConverterTests .cs

```diff
@@ -0,0 +1,117 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Globalization;
+using FluentAssertions;
+using Microsoft.CodeAnalysis.Sarif;
+using Microsoft.Sarif.Viewer.Converters;
+using Microsoft.Sarif.Viewer.Models;
+using Xunit;
+using System.Drawing;
```

Comment:
Sort the using directives as follows: Right click anywhere within the using directives, and select menu item Organize usings, Remove and sort usings.